src/cincinnati/mod.rs

```diff
@@ -20,6 +21,13 @@ pub static SCHEME_KEY: &str = "org.fedoraproject.coreos.scheme";
 /// Metadata value for "checksum" payload scheme.
 pub static CHECKSUM_SCHEME: &str = "checksum";
 
+lazy_static::lazy_static! {
+    static ref GRAPH_NODES: IntGauge = register_int_gauge!(opts!(
+        "zincati_cincinnati_graph_nodes",
```

Comment:
`_nodes` is the unit here, see https://prometheus.io/docs/practices/naming/#metric-names. An equivalent form would be `graph_nodes_count`, would you prefer that?

Reply:
Maybe `_total` to match their example of `http_requests_total`?

Reply:
`_total` is usually meant for accumulating counts over time. In this case it would hint at something like "the sum of nodes over all responses so far".

Reply:
`_total` is usually meant for an accumulating counter, monotonically increasing (i.e. here something like the sum over all responses so far).

I dragged in @s-urbaniak to tie-break this, and his recommendation for this gauge is `_graph_nodes_count`.